appserver/admingui/jdbc/src/main/resources/jdbcResourceEditTabs.inc

```diff
@@ -0,0 +1,109 @@
+        }
+        if(!#{pageSession.isAppScopedRes}){
+            if(#{pageSession.showTargetSection}) {
+                setPageSessionAttribute(key="useString" value="#{true}");
+                setPageSessionAttribute(key="useCheckBox" value="#{false}");
+                setPageSessionAttribute(key="has2Objects" value="#{false}");
+                setPageSessionAttribute(key="showDefaultButton" value="#{true}" );
+            }
+            if(!#{pageSession.showTargetSection}) {
+                setPageSessionAttribute(key="useString" value="#{false}");
+                setPageSessionAttribute(key="useCheckBox" value="#{true}");
+                setPageSessionAttribute(key="parentUrl2", value="#{sessionScope.REST_URL}/servers/server/server");
+                setPageSessionAttribute(key="childType2", value="resource-ref");
+  
+                setPageSessionAttribute(key="has2Objects" value="#{true}");
+                setPageSessionAttribute(key="showDefaultButton" value="#{true}" );
+                setPageSessionAttribute(key="selfUrl2", value="#{pageSession.parentUrl2}/#{pageSession.childType2}/#{pageSession.encodedName}");
+                gf.checkIfEndPointExist(endpoint="#{pageSession.selfUrl2}" exists="#{requestScope.hasAssociatedTarget}");
+                
+                if (#{hasAssociatedTarget}) {
+                    gf.getEntityAttrs(endpoint="#{pageSession.selfUrl2}", valueMap="#{pageSession.valueMap2}");
+                    // if the resource itself is set to disabled, the real status should show disabled.
+                    if( "#{pageSession.valueMap.enabled} = false"){
+                        mapPut(map="#{pageSession.valueMap2}", key="enabled", value="#{pageSession.valueMap.enabled}");
+                    }
+                }
+                if (!#{hasAssociatedTarget}) {
+                    createMap(result="#{pageSession.valueMap2}");
+                    mapPut(map="#{pageSession.valueMap2}", key="enabled", value="#{pageSession.valueMap.enabled}");
+                }
+                if(#{reload}) {
+                    mapPut(map="#{pageSession.valueMap2}", key="enabled", value="#{status}");
+                }               
+            }
+        }
+    />
+    <sun:tab id="general" immediate="$boolean{true}" text="$resource{i18n.common.General}" >
+        <!command
+            setSessionAttribute(key="resEditTabs" value="general");
+            gf.redirect(page="#{generalPage}");
+        />
+    </sun:tab>
+    <sun:tab id="targetTab" rendered="$pageSession{showTargetSection}" immediate="$boolean{true}" text="$resource{i18n.common.TargetTab}" >
+        <!command
+            setSessionAttribute(key="resEditTabs" value="targetTab");
+            gf.redirect(page="#{targetPage}&generalPage=#{generalPage}");
+        />
+    </sun:tab>
+    <sun:tab id="propsTab" immediate="$boolean{true}" text="$resource{i18n.inst.tab.MicroProfileProperties}" >
```

Comment:
I'm incredibly confused by this page. Is it just a separate proxy page that allows the microprofile prefix to be appended to the properties? I'd probably prefer just to have to specify that prefix manually, or move this configuration into the same place as the secrets dir config (i.e. a new tab in the microprofile config settings).